app/src/lib/gravatar.ts

```diff
@@ -8,7 +8,7 @@ import { getDotComAPIEndpoint } from './api'
  * @param email The email address associated with a user
  * @param size The size (in pixels) of the avatar to render
  */
-export function generateGravatarUrl(email: string, size: number = 200): string {
+export function generateGravatarUrl(email: string, size: number = 60): string {
```

Comment:
I'm a fan of defaults, but I was under the impression that it was determined not use them. I know this was already the case, but just curious if we can start using defaults in a function's parameters?

Reply:
@iAmWillShepherd I'm happy to review this decision on a case-by-case basis. Feel free to call this out the next time you encounter it on a PR.